app/javascript/contexts/auth/AuthProvider.jsx

```diff
@@ -26,6 +26,7 @@ export default function AuthProvider({ children }) {
     permissions: currentUser?.permissions,
     role: currentUser?.role,
     active: currentUser?.active,
+    signed_out: currentUser?.signed_out,
```

Comment:
We can clean this by introducing a small global state that includes the `currentUser` along with our small client state,
A client state is the state that the client application uses to store data that it needs to retrieve and has no relation with the server data (that includes as an example from where you were navigating, what component triggered the logic, your pointer coordination, scrolling coordination, ....)
So we won't do that hackery of `queryClient.setQueryData(...)`.